tests/API/Integration/BuildingRelationshipsTests.cs

```diff
@@ -82,14 +82,7 @@ public async Task CannotCreateMalformedBuildingRelationship()
 				Assert.Contains("The request body was malformed, the unitId and/or buildingId field was missing.", actual.Errors);
 				Assert.AreEqual("(none)", actual.Details);
 			}
-
```

Comment:
Instead of removing this test, can we modify it to test the new expected behavior?